qiskit/compiler/transpiler.py

```diff
@@ -191,7 +191,7 @@ def transpile(  # pylint: disable=too-many-return-statements
             This can also be the external plugin name to use for the ``routing`` stage.
             You can see a list of installed plugins by using :func:`~.list_stage_plugins` with
             ``"routing"`` for the ``stage_name`` argument.
-        translation_method: Name of translation pass ('unroller', 'translator', 'synthesis')
+        translation_method: Name of translation pass ('translator', 'synthesis')
```

Comment:
While we're at it, we could make it consistent with the rest of the docstring
```suggestion
        translation_method: Name of translation pass (``"translator"`` or ``"synthesis"``)
```

Reply:
Done!